keyboards/keyprez/rhino/rhino.c

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2021 Christian Sandven
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "rhino.h"
```

Comment:
```suggestion
#include "rhino.h"

bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    tap_code(clockwise ? KC_VOLU : KC_VOLD);
    return true;
}
```

Reply:
Just to understand: if I create a keymap later with different encoder functionality, it will override the volu/vold code?

Reply:
Yes and no.   If the user function returns false, then yes, it will replace the functionality.

The upside to this way, is that encoder support will work on the configurator.

Reply:
After I pushed this change, it seems to not register the encoder anymore. Also got an error that it could not find the encoder_update_user, that is why I tried the `ifndef` ,method. I am sure I am doing something wrong. Will look at it more in the weekend.

Reply:
It works fine if you're up to date. but your repo doesn't include the latest breaking changes code, which includes a number of encoder changes.